style.css

```diff
@@ -0,0 +1,1110 @@
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  /* stylelint-disable-next-line unit-no-unknown */
+  width: 100vw;
+  padding: 0%;
+  margin: 0%;
+  overflow-x: hidden;
+}
+
+.toolbar {
+  display: flex;
+  justify-content: space-between;
+  background: #3c3a39;
+  height: 72px;
+}
+
+.desktop-version {
+  display: none;
+}
+
+.toolbar img {
+  margin-right: 38.5px;
+  margin-top: 26px;
+}
+
+.name {
+  color: white;

```

Comment:
- [ ] Please, consider that the links are not available on the desktop version, as previous reviewer mentioned. Try to fix it 👍 
<img width="1436" alt="Screen Shot 2022-03-31 at 10 18 27" src="https://user-images.githubusercontent.com/49064106/161007642-7e2111af-ae22-4dc5-b0b4-741ab8dcce6e.png">